pkgs/applications/misc/someblocks/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchFromSourcehut, writeText, patches ? [ ], conf ? null }:
+stdenv.mkDerivation rec {
+  pname = "someblocks";
+  version = "1.0.1";
+
+  src = fetchFromSourcehut {
+    owner = "~raphi";
+    repo = "someblocks";
+    rev = version;
+    sha256 = "sha256-pUdiEyhqLx3aMjN0D0y0ykeXF3qjJO0mM8j1gLIf+ww=";
+  };
+
+  inherit patches;
+  postPatch =
+    let
+      configFile =
+        if lib.isDerivation conf || builtins.isPath conf
+        then conf else writeText "blocks.def.h" conf;
+    in
+    lib.optionalString (conf != null) "cp ${configFile} blocks.def.h";
+
+  dontConfigure = true;
+
+  NIX_CFLAGS_COMPILE = [ "-Wno-unused-result" ];
+
+  installPhase = ''
+    make install PREFIX="$out"
+  '';
```

I think this would be better expressed by setting `$PREFIX` like this:
```suggestion
installFlags = [ "PREFIX=$(out)" ];
```
because nix already knows to call `make install`.

---

I tried that first, but it didn’t work - make just tried to create /usr/local.

---

I think I might have done it wrong though, I’ll try again tomorrow.

---

Okay, done!